test/data/onion_upgradable.html

```diff
@@ -0,0 +1,12 @@
+<html>
+<head><title>OK</title></head>
+
+<meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">
```

Comment:
it should actually upgrade without this policy: https://blog.chromium.org/2019/10/no-more-mixed-messages-about-https.html

> In Chrome 81, mixed images will be autoupgraded to https://, and Chrome will block them by default if they fail to load over https://.